test.cpp

```
@@ -0,0 +1,132 @@
+        cout << "Grade: " << students[i].grade << "\n";
+        cout << "Gender: " << students[i].gender << "\n";
+        cout << "----------------------\n";
+    }
+}
+
+float calculateAverage(Student* students, int count) {
+    float sum;
+    for (int i = 0; i < count; i++) {
+        sum += students[i].grade;
+    }
+    return sum / count;
```

Cppcheck (2.10-2)

[warning] Uninitialized variable

(uninitvar)